source/utilities.tex

```diff
@@ -7093,9 +7093,9 @@
   template<class T1, class D1, class T2, class D2>
     requires @\libconcept{three_way_comparable_with}@<typename unique_ptr<T1, D1>::pointer,
                                        typename unique_ptr<T2, D2>::pointer>
-    compare_three_way_result_t<typename unique_ptr<T1, D1>::pointer,
-                               typename unique_ptr<T2, D2>::pointer>
-      operator<=>(const unique_ptr<T1, D1>& x, const unique_ptr<T2, D2>& y);
+      compare_three_way_result_t<typename unique_ptr<T1, D1>::pointer,
+                                 typename unique_ptr<T2, D2>::pointer>
+        operator<=>(const unique_ptr<T1, D1>& x, const unique_ptr<T2, D2>& y);
```

Comment:
This should be dropped, I believe. "requires" does not cause additional two-space indentation.
(same for the rest here)

Reply:
We're inconsistent about this.  If we don't want extra indents after requires, then we should fix the places that do indent.  I'm fine either way, although I have a slight preference to not indent also.  I've removed the extra indents from this patch for now.